### DISCUSSION TITLE
Can vs-extension Unocss show rem to px value like the following?

### DISCUSSION THREAD

**Discussion by cdx111** [unocss/unocss#2832] [Ideas]
the tailwind extension 
<img width="286" alt="image" src="https://github.com/unocss/unocss/assets/39432333/fb658143-7cb9-4c5e-a7be-dd9fb0bb7871">
the unocss extension
<img width="222" alt="image" src="https://github.com/unocss/unocss/assets/39432333/9212d361-54cf-434d-9f5d-28c9d7f000e0">

**Comment by zyyv** [comment]
#2808

**Reply by cdx111** [reply]
okay
